unpub/lib/src/static/index.html.dart

```diff
@@ -1696,11 +1696,35 @@ const content = """<!DOCTYPE html>
         background-color: transparent;
         border: 0;
       }
+      .banner {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            // background-color: #f5f5f5;
+            height: 40px;
+        }
+
+        .banner-text {
+            font-size: 14px;
+            font-weight: bold;
+            text-decoration: underline;
+            color: #00579b;
+        }
+
+        .banner-link {
+            text-decoration: none;
+        }
     </style>
     <link rel="icon" type="image/png" href="https://pub.dev/favicon.ico" />
     <script defer src="main.dart.js"></script>
   </head>
   <body>
+        <div class="banner">
```

Comment:
3. Add an HTML banner on everypage of unpub so it's easy to find the Flutter web app from main unpub UI